### DISCUSSION TITLE
(Suggestions) : Add SFX Editor with waveform editor copy/paste feature | 0.90 version

### DISCUSSION THREAD

**Discussion by MasterNama2008** [nesbox/TIC-80#1291] [Ideas]
I found the issue here: https://github.com/nesbox/TIC-80/issues/1159

Is no **waveform button**?
![image](https://user-images.githubusercontent.com/71924055/103443236-2891c280-4c83-11eb-83e3-79cb2e95ad01.png)

**Comment by darltrash** [comment]
I think you can just change the waveform from the green box here by dragging your cursor over it
